Sources/MapboxCoreNavigation/NavigationStatus.swift

```diff
@@ -0,0 +1,9 @@
+import Foundation
+import MapboxNavigationNative
+
+extension NavigationStatus {
+    /// Legacy `roadName` property that returns first road name based on the `roads` array.
+    var roadName: String {
+        roads.map({ $0.text }).prefix(while: { $0 != "/" }).joined(separator: " ")
```

Comment:
This implementation is what NN docs say in case we want the removed `roadName` back.
But do we want to have a different implementation here?

Reply:
I guess we can work on this later. I've tested that this function works and we get the same road name.